ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/ClusterPageHeader.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Flex, Skeleton, Title, LabelGroup, Label } from '@patternfly/react-core';
+
+import { gql } from '@apollo/client';
+import { getDateTime } from 'utils/dateUtils';
+
+export const clusterMetadataFragment = gql`
+    fragment ClusterMetadata on Cluster {
+        id
+        name
+        status {
+            orchestratorMetadata {
+                buildDate
+                version
+            }
+        }
+    }
+`;
+
+export type ClusterMetadata = {
+    id: string;
+    name: string;
+    status?: {
+        orchestratorMetadata?: {
+            buildDate?: string; // ISO 8601 date format
+            version: string;
```

Comment:
Please confine with BE eng that version will always be returned from the API; I seem to remember that we don't always know the version of a cluster at runtime.

Reply:
TIL

I'd hope that the API returns an empty string in this case, since the API declares it as non-nullable, but I left a [comment in the API doc](https://docs.google.com/document/d/1ljs0XYpn30d8I4tOE5euAmjcex_hpZqrjdUIoXndg8A/edit?disco=AAABLHWLCUU) to clarify.

Reply:
https://www.goodreads.com/book/show/66863.Only_the_Paranoid_Survive

Reply:
I made a note in the API doc and pinged the BE team to verify whether or not this value will always be present. I'll merge as-is now, since we are checking truthy-ness of the value before rendering in the place where it is used here.